### ISSUE TITLE
Hide the fold number

### ISSUE THREAD

**Issue by DeeGeeMan** [AstroNvim/AstroNvim#1062]
### Question

How do I hide the fold number in the column? I am using nvim 0.8.

Example:
<img width="1002" alt="fold-numbes" src="https://user-images.githubusercontent.com/21670040/194779265-a90a99ad-116c-4f15-967d-3f4691792082.png">

**Comment by mehalter** [comment]
Check out the option of `foldcolumn`. You can do this with `:h foldcolumn`. You probably are setting this somewhere or some plugin is doing it because the default in astronvim is `foldcolumn=0`.

**Comment by DeeGeeMan** [comment]
I am using [nvim-ufo](https://github.com/kevinhwang91/nvim-ufo), and it's not showing numbers.

Example:
<img width="1050" alt="nvim-ufo" src="https://user-images.githubusercontent.com/21670040/194779849-afa6e884-184c-4e88-9f6a-f32474706288.png">

**Comment by SunHuawei** [comment]
@DeeGeeMan , that's really cool! Would you please share your config?

**Comment by i-am-the-slime** [comment]
@DeeGeeMan I'm also curious how to configure `ufo` in astronvim.

**Comment by DeeGeeMan** [comment]
@SunHuawei @i-am-the-slime Update your `user/init.lua` with the following...
```
options = {
    opt = {
    -- Folding options
    foldenable = true,
    foldcolumn = "1",
    foldlevel = 99,
    foldlevelstart = 99,
    fillchars = [[eob: ,fold: ,foldopen:,foldsep: ,foldclose:]],
    }
}
plugins = {
    init = {
       { "kevinhwang91/nvim-ufo", requires = "kevinhwang91/promise-async" },
       ...
    }
} 
```
Then call `nvim +PackerSync` from the console to install the [nvim-ufo](https://github.com/kevinhwang91/nvim-ufo) plugin.

**Comment by SunHuawei** [comment]
@DeeGeeMan , thank you!
However, I tried your configuration but still can not see the icons on the left side.
![image](https://user-images.githubusercontent.com/11191857/200998264-7798b67c-c3d2-47a7-a3a1-2aa7f97d9da1.png)

I tried Option 2 and Option 3 of https://github.com/kevinhwang91/nvim-ufo#installation, still not working. Here is one of the changes.
![image](https://user-images.githubusercontent.com/11191857/200998142-fc9f5f19-e55d-42bf-aed8-0a4fbffe72c1.png)

BTW, I'm using nvim 0.8, should I remap `zR` and `zM`?

**Comment by SunHuawei** [comment]
@DeeGeeMan, I get it. I created a fold with `zf}` and then I can see the fold icon. Anyway, I don't have to remap `zR` and `zM`.

However, what I'm looking for is something to create folds for each possible code block. Does anyone know anything like this?

**Comment by fynnfluegge** [comment]
> Check out the option of `foldcolumn`. You can do this with `:h foldcolumn`. You probably are setting this somewhere or some plugin is doing it because the default in astronvim is `foldcolumn=0`.

@mehalter I face the same issue when I disable heirline somehow. Seems that the foldcolumn default is 1 without heirline. I disabled any of my custom plugins. In `options.lua` I found this:
```lua
foldcolumn = vim.fn.has "nvim-0.9" == 1 and "1" or nil, -- show foldcolumn in nvim 0.9
```
which means foldcolumn is enabled by default. Unfortunately I am not able to overwrite this on startup in my user config because it gets overwritten again. 
Long story short: Shouldn't this may be `foldcolumn = "0"` here? Then foldcolumn is dsiabled by default in any case and it can be toggled if needed. I really don't like the foldcolumn and it is annoying to always toggle it to 0 on startup 😄
